mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/definition/annotation/IndexedEmbedded.java

```diff
@@ -135,6 +136,9 @@
 	 */
 	String[] includePaths() default {};
 
+	@Incubating
+	String[] excludePaths() default {};
```

Comment:
Needs javadoc. I know, it's just a draft, but more importantly: keep in mind that the documentation (javadoc and reference) for `includeDepths` might need adjustments as well.